src/split-panel/styles.scss

```diff
@@ -198,14 +199,16 @@ $app-layout-drawer-width: calc(#{awsui.$space-layout-toggle-diameter} + 2 * #{aw
 }
 
 .pane-header-wrapper-bottom {
-  /* stylelint-disable-next-line plugin/no-unsupported-browser-features */
-  position: sticky;
-  top: 0;
   display: flex;
   align-items: center;
   flex-direction: column;
   z-index: 1; // should be above .content-bottom
   padding: 0 awsui.$space-scaled-2x-xxxl;
+  &.sticky-header {
+    /* stylelint-disable-next-line plugin/no-unsupported-browser-features */
```

Comment:
This now works in the entire range of all browsers we support. Might need a follow-up to upgrade the rule